### PR TITLE
Release ISP v1.0.6 translations [SVCS-305] [SVCS-306] [SVCS-307] [SVCS-308] 

### PR DESCRIPTION
**Pull request by sheriefvt**
Refs: https://openscience.atlassian.net/browse/SVCS-305
Refs: https://openscience.atlassian.net/browse/SVCS-306
Refs: https://openscience.atlassian.net/browse/SVCS-307
Refs: https://openscience.atlassian.net/browse/SVCS-308 
 
## Purpose
Add new translations for release ISP v1.0.6

## Summary of changes
Updated on Production:

- Italian   

Promoted to Production:

- Korea   
- Marathi
- Russian 
- Slovenian 
- Hebrew
- Thai

New to Staging:

- Maylay 
- Georgian
- Greek
- Simplified Chinese (Hong Kong)
- Romanian

## Testing notes
This is how the Georgian flag looks in the language picker component. 
image

### CODE REVIEW

File: app/locales/lt/translations.js [hunk 1]
```diff
@@ -100,1007 +100,1007 @@ export default {
     "97": "97",
     "98": "98",
     "99": "99",
-    "Key": "Lithuanian Translation",
+    "Key": "Italian Translation",
```
Comment (abought) [inline app/locales/lt/translations.js]: This appears to be the translation for italian ([it](https://github.com/abought/isp/blob/4e53b1ea4d910ee28800bae9d6c3d0dc019bb2e4/app/components/language-picker/countries.js#L270)), but it has been copied into the translation file for lithuanian ([lt](https://github.com/abought/isp/blob/4e53b1ea4d910ee28800bae9d6c3d0dc019bb2e4/app/components/language-picker/countries.js#L310-L310)). Can you double-check this and make sure these files contain the intended translations?

Comment (sheriefvt) [reply app/locales/lt/translations.js]: sorry for this will fix

File: app/locales/zh/translations.js [hunk 2]
```diff
@@ -100,53 +100,53 @@ export default {
     "97": "97",
     "98": "98",
     "99": "99",
-    "Key": "Chinese Translation",
+    "Key": "Final HK Chinese",
```
Comment (abought) [inline app/locales/zh/translations.js]: There are two separate Chinese translations in the ISP translations folder, which makes me think that overwriting the existing chinese translation might not be what we want.

We may want to revert this file (use the original chinese translation), and create a separate `zh-HK` translation file for the Hong Kong version. Long Ze may be able to advise and make sure that *both* of the chinese translation versions look right.

**Review by abought — CHANGES_REQUESTED** [review]
Changes requested for two translations files. The list of countries approved for production looks good.

**Comment by sheriefvt** [comment]
@abought .. thanks for your feedback completed changes
🐼

**Comment by abought** [comment]
Before I review this, can you perform a quick check (eg with Longze) to make sure that the first 1-2 pages of the survey show up correctly in each of the two chinese variants? (zh vs zh-hk)

We are not responsible for QA of the full translation, but we have never used sub-locales before, and I'd like to be sure that this feature works as intended.

**Review by abought — APPROVED** [review]

**Pull request merged**